### ISSUE TITLE
Guidelines do repositório

### ISSUE THREAD

**Issue by Mazuh** [discentes-imd/IntegraTI-Web#22]
A produção de vocês precisa estar de mais fácil acesso ao público externo, ou seja, minimamente documentada. Sugiro observar o trabalho do nosso engenheiro no repositório da API: organização das demandas/issues usando a aba [project (do seu repo GitHub)](https://github.com/discentes-imd/IntegraTI-Web/projects/); revisar o [readme](https://github.com/discentes-imd/IntegraTI-Web/blob/master/README.md) (não sei, só parece precisar); e, **principalmente** dar uma olhada nas guidelines de contribuição pra todos que trabalham e futuramente trabalharão no repositório consigam codificar como se fossem uma única só pessoa.

Reparem que o arquivo de contribuição de vocês é sugerido ao usuário pelo próprio GitHub ao criar uma issue, dar fork etc. Agora mesmo, enquanto escrevo isso, há um aviso "antes de submeter essa isue, por favor revise as guidelines" e um link para o arquivo de vocês. 

Aqui está o [contributing de vocês](https://github.com/discentes-imd/IntegraTI-Web/blob/master/CONTRIBUTING.md) e o [nosso](https://github.com/discentes-imd/IntegraTI-API/blob/master/CONTRIBUTING.md). Fizemos umas atualizações menores nele. E o de vocês é uma cópia já antiga, e bem nada a ver com a programação TypeScript.

**Comment by weslenbreno** [comment]
Concordo, vou dar uma atualizada nisso. Vou me basear no de vocês para que possamos estar em sintonia, mudarei apenas as questões que são específicas para o desenvolvimento em Angular + Typescript.

**Comment by gellyviana** [comment]
@weslenbreno, vc conseguiu fazer a  atualização?
